sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelects.java

```diff
@@ -12,7 +12,7 @@
  * @see SettingSelector
  * @see ConfigurationAsyncClient
  */
-public enum SettingFields {
+public enum SettingSelects {
```

Comment:
Let's change this from being an `enum` to `ExpandableStringEnum` to help make this version independent.